src/theme/index.ts

```diff
@@ -19,9 +25,15 @@ export const defaultTheme: Theme = {
   color: {
     background: {
       body: '#6448b3',
+      bottomMenuItem: '#fff',
+      loadingLogo: '#fff',
+      buttonPrimary: '#c63bea',
     },
     text: {
       body: '#fff',
+      bottomMenuItem: '#000',
+      loadingLogo: '#997cfa',
+      buttonPrimary: '#fff',
```

Comment:
Just a note for later to implement light / dark mode

Reply:
theming works oob so any other theme object passed would work 👍

Reply:
ty